test/isRepositoryEndpoint.test.ts

```diff
@@ -0,0 +1,16 @@
+import test from "ava";
+
+import * as prismic from "../src";
+
+test("returns true if the input is a repository endpoint", (t) => {
+	t.true(prismic.isRepositoryEndpoint("https://qwerty.cdn.prismic.io/api/v2"));
+});
```

Comment:
Since we want to accept any URL as a potential valid repository endpoint I'd add another assertion testing a random URL to prevent unintentional regressions (where one might want to refactor to only return `true` with the `prismic.io` domain).